worldwind-tutorials/src/commonMain/kotlin/earth.worldwind.tutorials/PathsTutorial.kt

```diff
@@ -21,7 +21,11 @@ class PathsTutorial(private val engine: WorldWind) : AbstractTutorial() {
                     Position.fromDegrees(30.0, -100.0, 1e6),
                     Position.fromDegrees(50.0, -40.0, 1e5)
                 )
-            )
+            ).apply {
+                attributes.apply {
```

Comment:
Do you need nested apply?
won't attributes.outlineWidth suffice?

Reply:
It will suffice, maybe this change should be removed there's already line in this tutorial with width greater than 1